engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/details/erratum-content-hosts.controller.js

```diff
@@ -59,17 +61,22 @@ angular.module('Bastion.errata').controller('ErratumContentHostsController',
                 searchStatements, errataIds;
 
             if ($scope.errata) {
+
                 errataIds = [$scope.errata['errata_id']];
             } else {
                 errataIds = _.map($scope.table.getSelected(), 'errata_id');
             }
 
             searchStatements = _.map(errataIds, function(errataId) {
-                return searchTerm + ' = "' + errataId + '"';
+                if(errataId) {
+                    return searchTerm + ' = "' + errataId + '"';
+                }
             });
             return searchStatements.join(" or ");
         };
 
+        nutupane.refresh();
+
         $scope.selectEnvironment = function (environmentId) {
             $scope.environmentId = environmentId;
             nutupane.refresh();
```

Comment:
i don't think you meant to include this here. this is whats causing a failure :)